.github/workflows/test-dapp-otc.yml

```diff
@@ -0,0 +1,82 @@
+name: Test Dapp OTC Desk
+
+# run CI on pushes to master, and on all PRs (even the ones that target other
+# branches)
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+
+jobs:
+  test-dapp:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['14.x']
+
+    steps:
+
+    - name: Checkout agoric-sdk
+      uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: cache node modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/yarn
+        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+
+    # Select a branch on dapp to test against by adding text to the body of the
+    # pull request. For example: #dapp-encouragement-branch: zoe-release-0.7.0
```

Comment:
```suggestion
    # pull request. For example: #dapp-otc-branch: zoe-release-0.7.0
```

Reply:
I decided not to change this example, since it would have to be changed in each file, and there are going to be about 8 of them. Do you think it needs to be changed?

Reply:
not a big deal. It's fine to leave it as is.